src/yaml_configs/customization_loader.rs

```diff
@@ -57,6 +60,32 @@ pub struct CodeLensCommand {
     pub messages: Vec<ChatMessage>,
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct CustomCMDLineTool {
+    pub description: String,
+    pub parameters: Vec<AtParamDict>,
+    pub parameters_required: Vec<String>,
+    pub command: String,
+    pub runs_in_background: bool,
+    #[serde(default)]
+    pub runs_in_background_false_timeout: usize,
+    #[serde(default)]
+    pub experimental: bool
```

Comment:
Don't need experimental, it's the user who creates the commands for themselves, not we create a tool that appears with --experimental flag.